### PR TITLE
fix(navigation): Fix not showing active tab 

### PR DESCRIPTION
**Pull request by mmelko**
Adds possibility to render html code in help (from #327) 
image

fixes: #267 #327

### CODE REVIEW

**Comment by mmelko** [comment]
@tadayosi if you approve this,  maybe it'd be good not to squash commits for preserving the history.  fix for #327 has two independent commits.

**Review by tadayosi — COMMENTED** [review]
I'm reluctant to fix the second issue now. Instead, we can just remove the html markers from any help.md we have now.

**Comment by mmelko** [comment]
> I'm reluctant to fix the second issue now. Instead, we can just remove the html markers from any help.md we have now.

Sure I actually agree. I will revert that one

File: packages/hawtio/src/help/HawtioHelp.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const HawtioHelp: React.FunctionComponent = () => {
           <Nav aria-label='Nav' variant='tertiary'>
             <NavList>
               {helpRegistry.getHelps().map(help => (
-                <NavItem key={help.id} isActive={location.pathname === help.id}>
+                <NavItem key={help.id} isActive={location.pathname.replace('/help/', '') === help.id}>
```
[the remark this answers is not in the document]
Comment (tadayosi) [reply packages/hawtio/src/help/HawtioHelp.tsx:34]: I used to do like this in JMX plugin, but it's somehow gone in the commit log.
```
  const path = (id: string) => `/jmx/${id}`

  const mbeanNav = (
    <Nav aria-label="MBean Nav" variant="tertiary">
      <NavList>
        {navItems.map(nav =>
          <NavItem key={nav.id} isActive={pathname === path(nav.id)}>
            <NavLink to={{ pathname: path(nav.id), search }}>{nav.title}</NavLink>
          </NavItem>
        )}
      </NavList>
    </Nav>
  )
```

**Comment by mmelko** [comment]
@tadayosi updated.

**Review by tadayosi — APPROVED** [review]
Please fix the same issue with JMX later.

**Pull request merged**